ui/src/components/AutoComplete2/Options/OptionList/OptionList.js

```diff
@@ -18,20 +18,23 @@ const OptionList = ({
   onSelectOption,
   deselectOption,
   children
-}) => (
-  <div className={styles.optionList}>
-    <InfiniteOptionList
-      options={options}
-      optionCount={optionCount}
-      fetchMore={fetchMore}
-      displayCount={displayCount}
-      rowHeight={rowHeight}
-      renderOption={renderOption}
-      onSelectOption={onSelectOption}
-      deselectOption={deselectOption} />
-    { children && <Divider /> }
-    { children }
-  </div>
-);
+}) => {
+  const out = (
```

Comment:
Sorry, I might change to function returning directly.
Is there a reason of changing this?

We can also write like below.

```
}) => {
  return (
    ...
  );
}
```

Reply:
Linting rules don't allow single statements in curly braces, which makes i a pain for debugging.

Reply:
I see.